github/utils.go

```diff
@@ -92,7 +92,11 @@ func parseRepoFullName(fullName string) (string, string) {
 func convertTimestamp(ctx context.Context, input *transform.TransformData) (interface{}, error) {
 	switch t := input.Value.(type) {
 	case *github.Timestamp:
-		return t.Format(time.RFC3339), nil
+		if t == nil {
```

Comment:
Open for better suggestions here. It can be `nil` so would be good to return `null` instead of fail the query

Reply:
@japborst I believe that the default in the switch case takes care of the `nil` case.

However, can you think of any situation wherein the value of `t` would be `nil`?

Reply:
I had to add this, so the case definitely exists 😄 

What happened was the following:
- The field was not present in the response
- The transform returns `nil`, passing to this function (from `transform.FromField("CreatedAt").Transform(convertTimestamp)`)

An alternative could be duplicate table definitions, but this seemed pretty sensible to support.

Reply:
Just doing `NullIfZero()` already did the trick, thanks 🙏